package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yt-channel-info",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Comment:
Are we following semver for this package?
If so this should be `3.0.0` due to breaking changes